src/landmask.jl

```diff
@@ -69,14 +69,16 @@ end
 """
     remove_landmask(landmask, ice_mask)
 
-Find the pixel indexes that are floating ice rather than soft or land ice. Returns an array of pixel indexes. 
+Apply the landmask to the ice mask to remove labeled ice pixels that overlap with the landmask. Returns
+a list of indices of pixels that are likely containing sea ice.
 
 # Arguments
 - `landmask`: bitmatrix landmask for region of interest
 - `ice_mask`: bitmatrix with ones equal to ice, zeros otherwise
 
 """
 ## NOTE(tjd): This function is called in `find_ice_labels.jl`
+## NOTE(dmw): For consistency, it would make sense to reverse the order of inputs to match landmask
 function remove_landmask(landmask::BitMatrix, ice_mask::BitMatrix)::Array{Int64}
```

Comment:
```
## NOTE(dmw): For consistency, it would make sense to reverse the order of inputs to match landmask
function remove_landmask(landmask::BitMatrix, ice_mask::BitMatrix)::Array{Int64}
```

Consider adding an issue.

Reply:
That's a good idea!